lab/lab2/js/part2-app-state.js

```diff
@@ -82,10 +98,14 @@ var Stamen_TonerLite = L.tileLayer('http://stamen-tiles-{s}.a.ssl.fastly.net/ton
 /* =====================
  CODE EXECUTED HERE!
 ===================== */
-
 downloadData.done(function(data) {
   var parsed = parseData(data);
+  //filterd version
+  filteredData = _.filter(parsed, function(obj){
+    return obj.District == 1;
+  });
   var markers = makeMarkers(parsed);
+  var filteredMarkers = makeMarkers(filteredData);
   plotMarkers(markers);
-  removeMarkers(markers);
+  removeMarkers(filteredMarkers);
```

Comment:
you plotted `markers` but then removed `filteredMarkers` so it doesnt actually remove anything. This is very close, but doesn't quite work.